frontend/src/components/helper/use-project-list.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from "@tanstack/react-query";
+import { apiClient } from "~/api/client";
+
+const TWO_MINUTES = 2 * 60 * 1000;
+
+export function useProjectList() {
+  return useQuery({
+    queryKey: ["PROJECT_LIST"],
```

Comment:
```suggestion
    queryKey: projectKeys.list,
```
use the key in key-factories.

Reply:
Also move `use-project-list.tsx` into `lib/hooks/` folder